json/Makefile

```diff
@@ -179,13 +179,16 @@ json-scala/target/application.jar:
 	$(MAKE) -C json-scala target/application.jar
 
 simdjson-dir := target/simdjson
+
+# We specify the simdjson commit to ensure reproducible builds.
 $(simdjson-dir): | target
 	$(GIT_CLONE) "https://github.com/simdjson/simdjson.git" $@ \
 	&& cd $@/singleheader \
+	&& git checkout bb2bc98a22b68471f193a3f3a60effd9d15183e2 \
```

Comment:
It's intended to verify the HEAD otherwise we could got stuck with the outdated version and it obviously won't be fair. I already have to verify cargo and maven dependencies manually, and I would prefer to avoid the manual verification of github dependencies. If HEAD is broken though, it's totally fine, but in that case please add a comment to the Makefile that it's not usable.

Reply:
Ok. That's fair enough.

I would personally prefer to benchmark specific upstream code, so the results are more predictable, but that's your code.

Changed.